<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManageDialog</class>
 <widget class="QDialog" name="BookManageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;等线&quot;;</string>
   </property>
   <property name="title">
    <string>库存图书</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>451</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>219</width>
       <height>449</height>
      </rect>
     </property>
     <widget class="QListWidget" name="list_book">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;等线&quot;;
background-image: url(:/background/resourse/b_W.png);
border:2px solid gray;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>311</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;等线&quot;;</string>
   </property>
   <property name="title">
    <string>详细信息</string>
   </property>
   <widget class="QPushButton" name="Button_confirm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	background-image: url(:/background/resourse/button.png);
}
QPushButton:hover{
	background-image: url(:/background/resourse/button_h.png);
}
QPushButton:pressed{	
	background-image: url(:/background/resourse/button_P.png);
}
</string>
    </property>
    <property name="text">
     <string>确认修改</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="Edit_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255,0.6,0.6);
border:2px solid gray;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图书简介</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_title">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>62</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_index">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_department">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>计算机学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>理学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>马克思主义学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>人文学院</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>信息与通信技术学院</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="Edit_author">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>102</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_right">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>任何人</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>教职工</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>学生</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="Edit_amount">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>150</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图书编号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>标题</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图书部门</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>作者</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>库存数量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图书权限</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_delete">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	background-image: url(:/background/resourse/button.png);
}
QPushButton:hover{
	background-image: url(:/background/resourse/button_h.png);
}
QPushButton:pressed{	
	background-image: url(:/background/resourse/button_P.png);
}
</string>
    </property>
    <property name="text">
     <string>删除图书</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/background/resourse/books5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_8</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
